### PR TITLE
Add update user profile endpoint

### PR DESCRIPTION
**Pull request by daallen303**

### CODE REVIEW

File: src/profile.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+**Optional Fields** (Note at least one field must be provided in the request)
+
+|      Field       |  Type  |                 Description                                                                        |
+| :--------------- | :----- | :------------------------------------------------------------------------------------------------- |
+| givenName        | String | First name                                                                                         |
+| familyName       | String | Last name                                                                                          |
+| featureUpdates   | String | Whether a user would like to receive emails for feature updates (valid values are true or false)   |
+| marketingUpdates | String | Whether a user would like to receive emails for marketing updates (valid values are true or false) |
```
Comment (ncjones) [inline src/profile.md:43]: these should be boolean

File: src/profile.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+
+## Updating a user's profile
+
+{% endpoint POST https://service.centrapay.com/api/merchants %}
```
Comment (ncjones) [inline src/profile.md:21]: wrong endpoint url

File: src/profile.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Profile
+nav_order: 6
+---
+
+# Profile
+{:.no_toc}
+
+A profile represents a centrapay user's attributes in the cognito user pool.
```
Comment (ncjones) [inline src/profile.md:10]: don't mention cognito

Comment (daallen303) [reply src/profile.md]: Removed the cognito reference

File: src/profile.md (hunk 2 above)
Comment (daallen303) [reply src/profile.md]: Fixed

File: src/profile.md (hunk 1 above)
Comment (daallen303) [reply src/profile.md]: Updated the docs to be boolean need to update the API, the decision for them to be strings was based on them being persisted as a string in cognito but boolean makes more sense.

File: src/profile.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+
+## Updating a user's profile
```
[the remark this answers is not in the document]
Comment (heymatthew) [reply src/profile.md:19]: Otherwise LGTM

**Review by heymatthew — APPROVED** [review]

File: src/profile.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (daallen303) [reply src/profile.md]: Added the experimental tag

**Pull request merged**